### PR TITLE
#126678329 Redesign Emails

### PR DESCRIPTION
**Pull request by NdagiStanley**
#### What does this PR do?
It makes the emails sent by Codango to be consistent.
#### Description of Task to be completed?
Have emails inherit from one template
#### How should this be manually tested?
The easiest way is by creating a session for Pair Programming and send an invite to an email you have access to then check the email.
#### Any background context you want to provide?
This is how different the email were:

> Popular posts

screenshot 2016-07-27 11 54 19
> Comments & Likes

screenshot 2016-07-27 11 54 38
> Session Invite

screenshot 2016-08-01 14 05 33
#### What are the relevant pivotal tracker stories?
126678329
#### Screenshots (if appropriate)
screenshot 2016-08-01 16 08 36

### CODE REVIEW

File: codango/pairprogram/views.py [hunk 1]
```diff
@@ -99,19 +101,28 @@ def send_invites(self, email, session, request):
             url = 'http://%s%s?session_id=%s' % (
```
Comment (andela-cobiora) [inline codango/pairprogram/views.py]: Be consistent with string formatting: Choose the `'{}'.format()` style over `'%s' %`

File: codango/pairprogram/views.py [hunk 2]
```diff
@@ -99,19 +101,28 @@ def send_invites(self, email, session, request):
...
+                'url': url,
+                'session_name': session.session_name
+            })),
+            html=loader.get_template(
+                'emails/session-invite.html'
+            ).render(Context({
+                'subject': 'Let\'s Start Pairing now!',
+                'url': url,
+                'session_name': session.session_name
+            }))
```
Comment (andela-cobiora) [inline codango/pairprogram/views.py]: Code repetition: the values passed into `loader.get_template` for both `text` and `html` are identical. Abstract it out to a single reusable variable. This way if you were to change it in the future, you would only need to change it in one place.

File: codango/resources/templates/emails/popular-post-updates.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{% for resource in popular_posts %}
+      {% if resource.snippet_text %}
+            {{ resource.snippet_text }}
+      {% endif %} 
+            {{resource.date_added}}
+            {{ resource.author }}
+
+      {% empty %}
+            No Popular Posts Today </p>
+  {% endfor %}
```
Comment (andela-cobiora) [inline codango/resources/templates/emails/popular-post-updates.txt]: Indentation for the `endfor` not in line with the corresponding opening `for`

File: codango/resources/templates/emails/popular-post-updates.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Hi,
+
+Here are some top posts you may have missed on Codango.
+{% for resource in popular_posts %}
+      {% if resource.snippet_text %}
+            {{ resource.snippet_text }}
+      {% endif %} 
+            {{resource.date_added}}
+            {{ resource.author }}
+
```
Comment (andela-cobiora) [inline codango/resources/templates/emails/popular-post-updates.txt]: Space: Any reason for this space?

File: codango/resources/templates/emails/popular-post-updates.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      {% empty %}
+            No Popular Posts Today </p>
```
Comment (andela-cobiora) [inline codango/resources/templates/emails/popular-post-updates.txt]: P tag Context: could you please explain this closing p tag?

File: codango/resources/templates/emails/popular-post-updates.txt (hunk 4 above)
Comment (NdagiStanley) [inline codango/resources/templates/emails/popular-post-updates.txt]: None at all.

**Pull request merged**